src/main/java/nl/tudelft/cse1110/andy/execution/metatest/implementations/LibraryMetaTest.java

```diff
@@ -0,0 +1,17 @@
+package nl.tudelft.cse1110.andy.execution.metatest.implementations;
+
+import nl.tudelft.cse1110.andy.execution.metatest.AbstractMetaTest;
+import nl.tudelft.cse1110.andy.execution.metatest.evaluators.MetaEvaluator;
+
+public class LibraryMetaTest extends AbstractMetaTest {
```

Comment:
What is a "LibraryMetaTest"? I don't get it by its name.

Reply:
A LibraryMetaTest is a meta test that replaces code in the library. This is the meta test type that corresponds to the already existing meta tests (inserting lines, replacing a substring, etc.)

Reply:
Ok, let's keep it like this. For a second, I thought of giving the name of what these meta tests do, in this case, they mutate the code under test. But let's see how it goes first before giving better names.